Formula/arks.rb

```diff
@@ -0,0 +1,33 @@
+class Arks < Formula
+  # cite Coombe_2018: "https://doi.org/10.1186/s12859-018-2243-x"
+  desc "Scaffold genome assemblies with 10x Genomics Chromium reads"
+  homepage "https://github.com/bcgsc/arks"
+  url "https://github.com/bcgsc/arks/archive/1.0.3.tar.gz"
+  sha256 "036971694e36de68d1285e385804489d478e7668459cd2fffccf9f64cf0facef"
+  head "https://github.com/bcgsc/arks.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "boost" => :build
+  depends_on "google-sparsehash" => :build
+  depends_on "gcc" if OS.mac? # for openmp
+  depends_on "zlib" unless OS.mac?
+
+  fails_with :clang # needs openmp
+
+  def install
+    inreplace "configure.ac", "AM_INIT_AUTOMAKE", "AM_INIT_AUTOMAKE(foreign)"
+    inreplace "autogen.sh", "automake -a", "automake -ac"
+    system "./autogen.sh"
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--with-boost=#{Formula["boost"].opt_include}"
+    system "make", "install"
+    prefix.install "Examples"
```

Comment:
You can modify `Makefile.am` so that `make install` installs these examples as well.

Reply:
As we discussed -- I'll modify `Makefile.am` in `master`, and I changed this line to `doc.install "Examples"`